polkadot/node/core/pvf/src/execute/queue.rs

```diff
@@ -236,7 +243,7 @@ impl Queue {
 		if eldest.waiting_since.elapsed() < MAX_KEEP_WAITING {
```

Comment:
We need to make it clear that this logic happens inside the same priority class wouldn't it ? 
It is not global anymore.